WcaOnRails/app/controllers/api/v0/wfc/dues_redirects_controller.rb

```diff
@@ -31,4 +31,10 @@ def create
       render json: wfc_dues_redirect.errors, status: :unprocessable_entity
     end
   end
+
+  def destroy
+    wfc_dues_redirect = WfcDuesRedirect.find(params[:id])
```

Comment:
Perhaps params.require will be safer?

Reply:
Oh yes, feeling bit embarrassing that I pointed out this in another PR very recently (in last 1 or 2 days), and I did that same mistake in my PR. 😅